WcaOnRails/app/views/competitions/_admin_index_table.html.erb

```diff
@@ -33,7 +33,7 @@
           <tr>
             <td>
               <i class="flag f16 <%= competition.country.iso2.downcase %>"></i><%= link_to competition.cellName, competition_path(competition) %>
-              <br /><strong><%= competition.country_name %></strong>, <%= competition.cityName %>
+              <br /><strong><%= competition.country.name_in(:en) %></strong>, <%= competition.cityName %>
```

Comment:
Because `country.name` is [localized](https://github.com/cubing/worldcubeassociation.org/blob/89f37a7c479063e5a1b789056e1b10e7ef40e216/WcaOnRails/app/models/country.rb#L9-L15), whereas the rest of this admin view is in English.
(I'm fine with displaying it localized, it would just be a bit weird)